src/sourmash_plugin_betterplot.py

```diff
@@ -855,6 +855,10 @@ def __init__(self, subparser):
             "--no-y-labels", action="store_true",
             help="disable Y axis labels"
         )
+        subparser.add_argument(
+            "--sort-by-category", action="store_true",
```

Comment:
```suggestion
            "--sort-rows-by-category", action="store_true",
```
You could imagine we might want to sort cols by category too ;)

Reply:
(so having this be `--sort-rows...` leaves that option open - not that we need to implement it now)